### PR TITLE
i18n(es): Update of `project-structure.md`, `routing.md`, `sharing-state.md`, `buddy.md`, `edgio.md` & `vercel.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description. 
Multiple updates as follows:
- `project-structure.md` based on this commit 79bdbb9ff909981606a62e276922a16bcd74e47c
- `routing.md` based on these commits c3b6a318f5c76fe817144c8760919ef24d2ecf35 & 79bdbb9ff909981606a62e276922a16bcd74e47c
- `sharing-state.md` based on this commit 36fd85a3d7051fc6b91be51e3847985452b70244
- `buddy.md` based on this commit b2eef6b76e2ccbaea7a0e84b25e73ff7ef21218e
- `edgio.md` based on this commit ce366938e7eef4e3aa8e03c0962661bac526ffa1
- `vercel.md` based on this commit e6f4c3c6f6db9756519619f64abffc451e71cee9

### CODE REVIEW

File: src/pages/es/core-concepts/routing.md [hunk 1]
```diff
@@ -104,7 +106,7 @@ const { path } = Astro.params;
 
 Esto generará `/sequences/uno/dos/tres`, `/sequences/cuatro` y `/sequences`. (Definir el parámetro restante como `undefined` permite emparejar con la página del nivel más alto.)
 
-Los parámetros rest pueden usarse con otros parámetros con nombre. Por ejemplo, podríamos representar el visor de archivos de GitHub con una ruta dinámica así:
+Los parámetros rest pueden usarse con **otros parámetros con nombre**. Por ejemplo, podríamos representar el visor de archivos de GitHub con una ruta dinámica así:
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/routing.md:109]:
```suggestion
Los parámetros rest pueden usarse con **otros parámetros nombrados**. Por ejemplo, podríamos representar el visor de archivos de GitHub con una ruta dinámica así:
```

File: src/pages/es/guides/deploy/edgio.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Puedes desplegar tu proyecto de Astro en [Edgio](https://www.edg.io/), una plata
 
 3. (Opcional) Habilita el Server Side Rendering.
 
-    Después de configurar [Server Side Rendering con Astro](/es/guides/server-side-rendering/), especifica la ruta del archivo del servidor en `edgio.config.js` como se muestra a continuación:
+    Después de configurar [Astro con @astrojs/node](/en/guides/integrations-guide/node/), especifica la ruta del archivo del servidor en `edgio.config.js` como se muestra a continuación:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/edgio.md:30]:
```suggestion
    Después de configurar [Astro con @astrojs/node](/es/guides/integrations-guide/node/), especifica la ruta del archivo del servidor en `edgio.config.js` como se muestra a continuación:
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias por otra PR @dreyfus92 🚀💜

File: src/pages/es/guides/deploy/edgio.md (hunk 2 above)
Comment (dreyfus92) [reply src/pages/es/guides/deploy/edgio.md:30]: Esa ruta no la encontré.

**Comment by dreyfus92** [comment]
Gracias @kevinzunigacuellar 💚

**Pull request merged**